src/styles/thread-list.scss

```diff
@@ -17,9 +17,25 @@
   }
 }
 
+.thread-list__card--theme-clean {
+  box-shadow: none;
+
+  &:hover {
+    box-shadow: none;
+  }
+}
+
 .thread-list__spacer {
   // This is a hidden element which is used to reserve space for off-screen
   // threads, so it should not occupy any space other than that set via its
   // 'height' inline style property.
   margin: 0;
 }
+
+.thread-list__separator--theme-clean {
+  border: 0;
+  border-top: 1px solid #E1E1E1;
+  margin-bottom: 10px;
+  margin-top: -8px;
+  width: 380px;
```

Comment:
The width of the separator is fixed here, which leads to what I guess are unexpected results if the sidebar is made wider by the user:

![screenshot 2017-10-20 17 24 13](https://user-images.githubusercontent.com/2458/31831485-99ad4f3c-b5bb-11e7-906c-2b7afbc8bdbf.png)